nrf_security/src/drivers/nrf_cc3xx/Kconfig

```diff
@@ -84,7 +142,7 @@ config PSA_CRYPTO_DRIVER_ALG_GCM_CC3XX
 	bool
 	prompt "PSA AES GCM support - cc3xx" if !PSA_PROMPTLESS
 	default y if !PSA_DEFAULT_OFF
-	depends on PSA_WANT_ALG_GCM
+	depends on PSA_WANT_ALG_GCM && !CRYPTOCELL_CC310_USABLE
```

Comment:
Why is it not `CRYPTOCELL_CC310_USABLE `, I would have understood it that way that we use the CC3XX config, if the cryptocell is usable

Reply:
This seems resolved to me

Reply:
Is it a typo? Was that intended to be !CRYPTOCELL_CC312_USABLE ?

Reply:
We support GCM on CC312 only (which is the inverse of !CRYPTOCELL_CC310_USABLE). But the negative is unneccesary, so changing this to CRYPTOCELL_CC312_USABLE

Reply:
Fixed